keanacobarde/on-paper
# NEW USER PATCH

## Description
- Addresses the issue of, when a brand new user signs in, they have no months within the FireBase, so the Timeline page is rendered non-functional and the display does not show a number. 
- Created a conditional API call which, if the difference between the last sign in time and the creation time is smaller than a singular millisecond, a collection of months associated with the user.uid are posted to the FireBase. 

## Related Issue
#48 - Addresses the first half of this that pertains to the bug. May create a separate issue ticket. 

## Motivation and Context
This allows for new users to access necessary functionality of the application.

## How Can This Be Tested?
-  Pull PR branch
- Install necessary dependencies
- Sign in with a new user account and check the timeline page. There should be no errors.

## Screenshots (if appropriate):
N / A

## Types of changes
